Makefile

```diff
@@ -0,0 +1,11 @@
+# $FreeBSD$
+
+TESTSDIR=	${LOCALBASE}/tests/tcptestsuite
+ATF_TESTS_SH=	tcptestsuite_atf_test
+
+CLEANFILES+=	tcptestsuite_atf_test.sh
+
+tcptestsuite_atf_test.sh:
+	/bin/sh ${.CURDIR}/make_atf.sh >$@
+
+.include <bsd.test.mk>
```

Comment:
I tested with the port change that I mentioned above.  It works just fine.  It creates the `Kyuafile` automatically, and the default `make install` target works.  I just have to create the directories first, since `mtree` doesn't.

Reply:
Cool!  I didn't realize that worked now.